source/gui/addonStoreGui/viewModels/store.py

```diff
@@ -648,12 +648,6 @@ def _cancelDownloadForAddon(self, listItemVM: AddonListItemVM[_AddonStoreModel])
 				if listItemVM == addon[0] and not future.cancel():
 					self._downloader.progress.pop(addon[0], None)
 
-		try:
-			# If the download was cancelled, the file may have been partially downloaded.
-			os.remove(listItemVM.model.tempDownloadPath)
-		except FileNotFoundError:
-			log.debug(f"File already removed {listItemVM.model.tempDownloadPath}")
```

Comment:
why was this removed

Reply:
Now the state after an add-on download is canceled no longer needs to take `tempDownloadPath` into account, and cleanup continues to be done by `_down`.